openedx/adg/lms/applications/api_urls.py

```diff
@@ -0,0 +1,17 @@
+"""
+All urls for applications APIs
+"""
+from django.conf.urls import include
+from django.urls import path
+from rest_framework.routers import DefaultRouter
+
+from .api_views import EducationViewSet, WorkExperienceViewSet
+
+router = DefaultRouter()
+router.register('education', EducationViewSet, basename='education')
+router.register('work_experience', WorkExperienceViewSet, basename='work_experience')
+
+app_name = 'applications'
```

Comment:
Are we using this app name anywhere?

Reply:
For providing namespace for urls we need to add this variable. I was getting an error to add app_name when using namespace.